agent/src/datafile.rs

```diff
@@ -339,9 +339,11 @@ impl DataFile {
 
         match running_snapshots.get_mut(&request.name) {
             None => {
+                // A read-only downstairs never existed for this snapshot,
```

Comment:
Is the name "running_snapshots" still correct here?
This could be a current snapshot or one that we already deleted, right?